tests/utils/test_uri.py

```diff
@@ -277,14 +278,14 @@ def test_is_databricks_acled_artifacts_uri():
     ('dbfs://@databricks:port/path/to/nowhere', 'databricks'),
     ('dbfs://profile@databricks/path/to/nowhere', 'databricks://profile'),
     ('dbfs://profile@databricks:port/path/to/nowhere', 'databricks://profile'),
-    ('dbfs://scope:key_prefix@databricks/path/abc', 'databricks://scope/key_prefix'),
-    ('dbfs://scope:key_prefix@databricks:port/path/abc', 'databricks://scope/key_prefix'),
+    ('dbfs://scope:key_prefix@databricks/path/abc', 'databricks://scope:key_prefix'),
+    ('dbfs://scope:key_prefix@databricks:port/path/abc', 'databricks://scope:key_prefix'),
     # Treats secret key prefixes with ":" to be valid
-    ('dbfs://incorrect:netloc:format@databricks/path/a', 'databricks://incorrect/netloc:format'),
+    ('dbfs://incorrect:netloc:format@databricks/path/a', 'databricks://incorrect:netloc:format'),
```

Comment:
we don't want to allow this anymore, right?

Reply:
i think for `test_remove_databricks_profile_info_from_artifact_uri` it's okay to remove the info in this invalid case.

Reply:
also should add the invalid case to `test_is_valid_dbfs_uri` below

Reply:
Agreed - I think the remove stuff probably doesn't need the validation